src/ensembl/src/content/app/browser/drawer/Drawer.tsx

```diff
@@ -15,12 +18,16 @@ import DrawerGC from './drawer-views/DrawerGC';
 
 import closeIcon from 'static/img/track-panel/close.svg';
 
+import { find } from 'lodash';
```

Comment:
By doing so, you import the whole lodash and then pick just one method off it (unless webpack has learned how to tree-shake it properly). And we are not using lodash-webpack-plugin that will remove unused lodash code. Don't do that. Import specific functions (`import find from 'lodash/find'`).

Reply:
That's done.